### DISCUSSION TITLE
Grandorgue on iPad

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#942] [Developers]
I ask if there is some forecast to make some porting of GO on IOS.
At present, only some good app do exist on Apple Store by Marcus Sigg. THey are good (i bought one and think i'll purchase some more) but afaik they're real application (based on framework Napo); they run syandalone and don't allow to load samplesets as GO.

**Comment by rousseldenis** [comment]
AFAIK, the apps developped for Ipad are totally different (Cocoa/C) as GrandOrgue is developped in C++. Nothing has been done yet (and nothing in the near future).

But @oleg68 and @larspalo should help you on the programming subject.

**Reply by paolo2504** [reply]
Thanks Roussel.

**Comment by oleg68** [comment]
GrandOrgue is heavy based on [wxWidgets graphical library](https://www.wxwidgets.org/), so it can be ported only to the platforms where wxWidgets are supported. It is easier to port other things (sound, midi etc) than the graphics.

For now wxWidgets don't work on iOS. There is [wxAndroid](https://wiki.wxwidgets.org/WxAndroid/docs) that makes porting to Android more probably than to iOS.

**Reply by paolo2504** [reply]
Oleg , appreciate your kind reply.
